system-tests/__snapshots__/component_testing_spec.ts.js

```diff
@@ -786,9 +786,9 @@ exports['experimentalSingleTabRunMode / executes all specs in a single tab'] = `
 
   (Screenshots)
 
-  -  /XXX/XXX/XXX/cypress/screenshots/1_fails.cy.js/simple failing spec -- fails (fai     (1280x720)
+  -  /XXX/XXX/XXX/cypress/screenshots/1_fails.cy.js/simple failing spec -- fails (fai     (1280x603)
      led).png                                                                                       
-  -  /XXX/XXX/XXX/cypress/screenshots/1_fails.cy.js/simple failing spec -- fails agai     (1280x720)
+  -  /XXX/XXX/XXX/cypress/screenshots/1_fails.cy.js/simple failing spec -- fails agai     (1280x603)
      n (failed).png                                                                                 
```

Comment:
Why did these resolutions change?

Reply:
There is a bug in chrome headless, in particular the new `headless=new` flag where `--window-size` is not set correctly. The issue is closed in chromium but multiple users are reporting issue https://bugs.chromium.org/p/chromium/issues/detail?id=1416398.